packages/integration/src/awsS3/config.ts

```diff
@@ -46,6 +48,16 @@ export type AwsS3IntegrationConfig = {
    * roleArn
    */
   roleArn?: string;
+
+  /**
+   * validateHost
+   */
+  validateHost: boolean;
```

Comment:
```suggestion
  validateHost?: boolean;
```

Since it's optional

Reply:
And suggestion since booleans tend to be semantically unclear and impossible to evolve:

Maybe make this `urlFormat?: 'aws' | 'common'` or something?

And one more thought - even if that new parameter did not exist, couldn't it just be assumed that if the host part ended in `.amazonaws.com`, then it's format A, otherwise format B?

Reply:
@OrkoHunter see my comment to `ssl` flag.

@freben I understand the only benefit of host validation would be to prevent usage of non-aws hosts in production configurations. I personally would prefer simplicity over extra config parameters as you are suggesting. LMK, and I'll make the change to remove `validateHost` from config. `ssl` flag is still needed though to indicate whether to use https for non-aws hosts...